src/Helper/Navigation/AbstractHelper.php

```diff
@@ -188,6 +188,10 @@ public function __toString()
     public function findActive($container, $minDepth = null, $maxDepth = -1)
     {
         $this->parseContainer($container);
+        if (null === $container) {
+            return [];
+        }
```

Comment:
This code change requires a corresponding test case addition

Reply:
Actually this is complicated to add test for this.
Because we technically can pass null as $container to `findActive` method, but we cannot create a test for this because PSALM will says "Argument 1 of Laminas\View\Helper\Navigation\Breadcrumbs::findActive cannot be null"

As PSALM will not support conditional param-out, it is the only way I've found to satisfy code behavior and PSALM checks